bin/linters.sh

```diff
@@ -133,7 +133,7 @@ function check_licenses() {
 }
 
 ensure_pilot_types
-format
+# format
```

Comment:
Here too

Reply:
Same reason as above. I'm not sure what the fate of circle+fmt is.